src/components/SlateEditor/plugins/SlateToolbar/index.js

```diff
@@ -32,14 +44,77 @@ export default function toolbarPlugin() {
               index,
             )
           }
+          onButtonClick={onButtonClick}
           name={name}
         />
       </>
     );
   };
 
+  const isBoldHotkey = isKeyHotkey('mod+b');
+  const isCodeHotKey = isKeyHotkey('mod+k');
+  const isConceptBlockHotKey = isCodeHotkey('mod+alt+c');
+  const isH2HotKey = isKeyHotkey('mod+2');
+  const isH3HotKey = isKeyHotkey('mod+3');
+  const isFootnoteHotKey = isCodeHotkey('mod+alt+f');
+  const isItalicHotKey = isKeyHotkey('mod+i');
+  const isLetteredListHotKey = isCodeHotkey('mod+alt+a');
+  const isLinkHotKey = isCodeHotkey('mod+alt+l');
+  const isListHotKey = isKeyHotkey('mod+l');
+  const isMathHotKey = isKeyHotkey('mod+m');
+  const isNumberedListHotKey = isCodeHotkey('mod+alt+1');
+  const isQuoteHotKey = isCodeHotkey('mod+alt+q');
```

Comment:
tydeligvis funker ikke `cmd+option+q` på mac heller for det også er en mulig måte å avslutte prosesser på. Siden det er en blockquote, kanskje du kan bruke `cmd+option+b`/`ctrl+alt+b`.

Reply:
Ja, endra til b